core/ir/x86/instr.c

```diff
@@ -63,32 +65,31 @@ instr_set_x86_mode(instr_t *instr, bool x86)
 bool
 instr_get_x86_mode(instr_t *instr)
 {
-    return TEST(INSTR_X86_MODE, instr->flags);
+    return (instr->isa_mode == DR_ISA_IA32);
 }
 #endif
 
 bool
 instr_set_isa_mode(instr_t *instr, dr_isa_mode_t mode)
 {
 #ifdef X64
-    if (mode == DR_ISA_IA32)
-        instr_set_x86_mode(instr, true);
-    else if (mode == DR_ISA_AMD64)
-        instr_set_x86_mode(instr, false);
-    else
+    if ((mode != DR_ISA_IA32) && (mode != DR_ISA_AMD64)) {
         return false;
+    }
 #else
-    if (mode != DR_ISA_IA32)
+    if (mode != DR_ISA_IA32) {
```

Comment:
Removed brackets from if-then-else single statements.
(I'm used to put them all the time, mostly because if I add instructions later on and I forget to add brackets it always takes me a while to figure out why my code isn't behaving as it should :) but I'll follow the coding style).